dev/devicelab/bin/tasks/platform_services__start_up.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
Let's remove this task. One task should be sufficient to test platform services. We can on-board more fine-grained tests after we teach our test infrastructure to scale in the future.